config-scripts/cups-compiler.m4

```diff
@@ -108,8 +108,8 @@ AS_IF([test -n "$GCC"], [
     ], [
 	# Otherwise use the Fortify enhancements to catch any unbounded
 	# string operations...
-	CFLAGS="$CFLAGS -D_FORTIFY_SOURCE=2"
-	CXXFLAGS="$CXXFLAGS -D_FORTIFY_SOURCE=2"
+	CFLAGS="$CFLAGS -D_FORTIFY_SOURCE=3"
+	CXXFLAGS="$CXXFLAGS -D_FORTIFY_SOURCE=3"
```

Comment:
Would it be possible to look for presence of `_FORTIFY_SOURCE` in the flags and then skipping addition if it's already present?  That way the flag won't end up overriding distribution flags.